client/branded/src/search-ui/components/RepoMetadata.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import classNames from 'classnames'
+
+import { Badge } from '@sourcegraph/wildcard'
+
+import styles from './RepoMetadata.module.scss'
+
+export const RepoMetadata: React.FunctionComponent<{
+    keyValuePairs: [string, string | null | undefined][]
+    className?: string
+    small?: boolean
+}> = ({ keyValuePairs, className, small }) => {
+    if (keyValuePairs.length === 0) {
+        return null
+    }
+    return (
+        <div className={classNames(styles.repoMetadata, className, 'd-flex align-items-center flex-wrap')}>
+            {keyValuePairs.map(([key, value]) => (
+                <span className="d-flex align-items-center justify-content-center" key={`${key}:${value}`}>
+                    <Badge small={small} variant="info" className={classNames({ [styles.repoMetadataKey]: value })}>
+                        {key}
+                    </Badge>
+                    <Badge small={small} variant="secondary" className={styles.repoMetadataValue}>
+                        {value}
+                    </Badge>
+                </span>
```

Comment:
You can instead use one `<Badge` with 2 children and then style just the background of the one you need to change the colour of, so something like:
```suggestion             
                <Badge small={small} variant="secondary" className={styles.repoMetadataKey} key={`${key}:${value}`}>
                    <span className="key">{key}<span>
                    <span className="value">{value}</span>
                </Badge>
```

Looks cleaner and IMO is a better resulting HTML structure. You also do not need to touch border radius at all this way.

Reply:
I don't think this is better than current version, as it will require resetting padding of inner child and setting bg color of one of them. I think using badge with variants (info, secondary) is more explicit.

Reply:
Good point, I've updated to use [description list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) HTML tags, as well as added aria attributes. Would appreciate the feedback.